packages/webassemblyjs/test/compile/fixtures/valide-module/exec.tjs

```diff
@@ -0,0 +1,11 @@
+it("should return return true on WebAssembly.validate", () => {
+  return WebAssembly.validate(wasmmodule).then(m => {
```

Comment:
Validate doesn't return a promise but a boolean.

Reply:
Oops, yes. So tjs doesn't work :)

Reply:
No worries, I'm currently fixiing it locally.

Reply:
Actually tjs is just a regular JavaScript file, which is executed by `make test`.

Reply:
Got it, thanks! 👍